docs/rest_api.md

```diff
@@ -236,25 +240,32 @@ Admins or the project that is the owner can list all attached nodes. Other proje
 
 Add <project> to access list for <network>.
 
-If the network or project does not exist a NotFoundError will be raised.
-
 Authorization requirements:
 
 Only admins or the network owner can grant a project access to the network.
 
+Possible Errors:
+
+* 400, if:
+   * the network or project does not exist
+   * the project already has access to the network
```

Comment:
Can you double check the status codes? I'm fairly certain the first is a 404, and I would guess the second is a 409.